posthog/models/raw_sessions/migrations.py

```diff
@@ -2,11 +2,13 @@
 
 from posthog.models.raw_sessions.sql import RAW_SESSIONS_DATA_TABLE, TABLE_BASE_NAME
 
+# perf
 ADD_PAGEVIEW_AUTOCAPTURE_SCREEN_UP_TO_2_COLUMN_SQL = """
 ALTER TABLE {table_name} on CLUSTER '{cluster}'
 ADD COLUMN IF NOT EXISTS
 page_screen_autocapture_uniq_up_to
 AggregateFunction(uniqUpTo(1), Nullable(UUID))
+AFTER maybe_has_session_replay
```

Comment:
This should have been included before - this is why I rearranged the fields in SQL.py to better match the order in prod

<img width="561" alt="Screenshot 2024-08-20 at 16 15 10" src="https://github.com/user-attachments/assets/74b31816-3ae9-4171-9924-c72e3d6e2261">